tests/e2e/machine_pool_test.go

```diff
@@ -21,36 +27,48 @@ var _ = Describe("TF Test", func() {
 		AfterEach(func() {
 			return
 		})
-		It("MachinePoolExampleNegative", func() {
-
-			mpParam := &EXE.MachinePoolArgs{
-				Token: token,
-				// OCMENV:      "staging",
-				Cluster:     cluster_id,
-				Replicas:    3,
-				MachineType: "invalid",
-				Name:        "testmp",
-			}
-
-			_, err := EXE.CreateMyTFMachinePool(mpParam, "-auto-approve", "-no-color")
-			Expect(err).To(HaveOccurred())
-			Expect(err.Error()).Should(ContainSubstring("is not supported for cloud provider 'aws'"))
-		})
-		It("MachinePoolExamplePositive", func() {
-			mpParam := &EXE.MachinePoolArgs{
-				Token: token,
-				// OCMENV:      "staging",
-				Cluster:     cluster_id,
-				Replicas:    3,
-				MachineType: "r5.xlarge",
-				Name:        "testmp3",
-			}
-
-			_, err := EXE.CreateMyTFMachinePool(mpParam, "-no-color")
-			Expect(err).ToNot(HaveOccurred())
-
-			err = EXE.DestroyMyTFMachinePool(mpParam, "-no-color")
-			Expect(err).ToNot(HaveOccurred())
+		// It("MachinePoolExampleNegative", func() {
+
+		// 	mpParam := &exe.MachinePoolArgs{
+		// 		Token: token,
+		// 		// OCMENV:      "staging",
+		// 		Cluster:     cluster_id,
+		// 		Replicas:    3,
+		// 		MachineType: "invalid",
+		// 		Name:        "testmp",
+		// 	}
+
+		// 	_, err := exe.CreateMyTFMachinePool(mpParam, "-auto-approve", "-no-color")
+		// 	Expect(err).To(HaveOccurred())
+		// 	Expect(err.Error()).Should(ContainSubstring("is not supported for cloud provider 'aws'"))
+		// })
+		Context("Author:amalykhi-High-OCP-64757 @OCP-64757 @amalykhi", func() {
+			It("Create a second machine pool", func() {
+				token := os.Getenv(con.TokenENVName)
+				replicas := 3
+				machine_type := "r5.xlarge"
+				name := "testmp14"
+				MachinePoolArgs := &exe.MachinePoolArgs{
+					Token: token,
+					//OCMENV: "staging",
+					Cluster:     cluster_id,
+					Replicas:    replicas,
+					MachineType: machine_type,
+					Name:        name,
+				}
+				mpService := exe.NewMachinePoolService(con.MachinePoolDir)
+				err := mpService.Create(MachinePoolArgs)
+				mp_out, err := mpService.Output()
```

Comment:
Let's add defer to clean up the created machinepool after testing

Reply:
added mp.Destroy
Is it fine @xueli181114 ?

Reply:
Yes